migtests/tests/import-file/validate

```diff
@@ -71,6 +83,30 @@ def file_import_done_checks(tgt):
 	print(f"t1_escape_backslash_data: {t1_escape_backslash_data}\nExpected data: {T1_ESCAPE_BACKSLASH_DATA}\n")
 	assert t1_escape_backslash_data == T1_ESCAPE_BACKSLASH_DATA
 
+	test_backspace_char_data = tgt.fetch_all_table_rows("test_backspace_char")
```

Comment:
see if this code can be reduced to a for loop, since we are doing same thing for each table. 
For example by defining all these tables in a list and run the loop for each to check